src/core/crypto.c

```diff
@@ -49,11 +48,21 @@ QuicCryptoDumpSendState(
         uint32_t i = 0;
         QUIC_SUBRANGE* Sack;
         while ((Sack = QuicRangeGetSafe(&Crypto->SparseAckRanges, i++)) != NULL) {
-            QuicTraceLogConnVerbose(CryptoDumpUnacked, Connection, "  unACKed: [%llu, %llu]", UnAcked, Sack->Low);
+            QuicTraceLogConnVerbose(
+                CryptoDumpUnacked,
+                Connection,
+                "  unACKed: [%llu, %llu]",
+                UnAcked,
+                Sack->Low);
             UnAcked = Sack->Low + Sack->Count;
         }
         if (UnAcked < (uint64_t)Crypto->MaxSentLength) {
-            QuicTraceLogConnVerbose(CryptoDumpUnacked, Connection, "  unACKed: [%llu, %u]", UnAcked, Crypto->MaxSentLength);
+            QuicTraceLogConnVerbose(
+                CryptoDumpUnacked,

```

Comment:
I say we have every trace point in the code use a unique name, at least modulo human error. Every time I'm investigating traces and find an error event that is logged from multiple places in the code, I wish it weren't the case.

Reply:
Yes, we have a few of these. I might make a pass in the future to see if I can de-dup things.

Reply:
@maolson-msft  - chk the feature/clog branch - thats what I've done.  you'll find lots of examples where the ID is obviously human generated.  I did this with WPP as well.  the idea is that before you take clog to master that we go through those and make them clean

in doing that you'll gain a bunch of abilities in tooling over what WPP gives